### ISSUE TITLE
[Enhancement] Please allow history to be placed on the tom menu of the main screen.

### ISSUE THREAD

**Issue by spitfire** [iv-org/invidious#4703]
**Is your enhancement request related to a problem? Please describe.**
Please allow history to be placed on the tom menu of the main screen. I use it more often than any of the other sections there, and every time I need to use it, I have to go through subscriptions.
Also for some reason that page does not display titles, descriptions, or anything else other than thumbnails:
<img width="1131" alt="Zrzut ekranu 2024-05-17 o 12 01 52" src="https://github.com/iv-org/invidious/assets/42872/2b0900a7-e84b-4229-9043-d6df249f9346">

unlike the other views:

<img width="1141" alt="Zrzut ekranu 2024-05-17 o 12 02 44" src="https://github.com/iv-org/invidious/assets/42872/24ccef65-c0b1-46f0-9882-732da5be992f">

**Describe the solution you'd like**
Please make History an option I can choose to display on the starting page:
![Uploading Zrzut ekranu 2024-05-17 o 12.03.04.png…]()

Please display information about the videos in the history view,, instead of just showing thumbnails.

**Comment by unixfox** [comment]
https://github.com/iv-org/invidious/issues/3035

**Comment by spitfire** [comment]
> https://github.com/iv-org/invidious/issues/3035

What about being able to place it on the main screen?
I don't care about "popular" videos and I rarely subscribe to anything but being able to go back to history quickly would be really useful to me.

**Comment by unixfox** [comment]
you mean having the history on the main page? mmh that's not really something we would like to offer as we would have to allow every single page to be made as the home page then.
but you can have a workaround for this. use a reverse proxy and make the `/` path to redirect to the history page.

**Comment by spitfire** [comment]
> you mean having the history on the main page? mmh that's not really something we would like to offer as we would have to allow every single page to be made as the home page then. but you can have a workaround for this. use a reverse proxy and make the `/` path to redirect to the history page.
No, I mean instead of one of these. I don't use the first two anyway.
<img width="1075" alt="Zrzut ekranu 2024-05-18 o 16 54 28" src="https://github.com/iv-org/invidious/assets/42872/67046442-f96a-4889-9e73-fa26bdd4d0a6">
In settings there is an option to change what they are:
<img width="969" alt="Zrzut ekranu 2024-05-18 o 16 55 59" src="https://github.com/iv-org/invidious/assets/42872/f960afa4-a776-409a-8e9c-8552d6b6f5c5">
but none of them allows me to choose history:
<img width="143" alt="Zrzut ekranu 2024-05-18 o 16 56 51" src="https://github.com/iv-org/invidious/assets/42872/d9b2d8e9-038a-4805-951d-f2639db9fabd">

**Comment by unixfox** [comment]
ha yes, we are not going to add every page there in this section sorry.

**Comment by spitfire** [comment]
> ha yes, we are not going to add every page there in this section sorry.



What other pages would that be? I think history (being able to quickly go back to stuff watched recently) is one of basic features of a video service frontend.

i.e. Youtube didn't add link to every single page and feature of it on a sidebar, but history is there (and it's close to the top by default), because it's useful:
 <img width="205" alt="Zrzut ekranu 2024-05-18 o 20 49 18" src="https://github.com/iv-org/invidious/assets/42872/807ab221-b6eb-4c0f-aecd-1862cfb6a4e5">

**Comment by SamantazFox** [comment]
> #3035

Reopening, as this is a completely different issue
